### ISSUE TITLE
Get status of radiobuttons and checkboxes?

### ISSUE THREAD

**Issue by Rapid1898-code** [py-pdf/pypdf#1334]
Hello - i try to get the status (checked / unchecked) of the radiobuttons and checkboxes in this document.

I tried it with the following code but I always get a `None` return

## Example

```python
from PyPDF2 import PdfReader

reader = PdfReader ("input.pdf")
fields = reader.get_fields()    
print(fields)
  ```

Find attached the pdf-file i want to read:
[input.pdf](https://github.com/py-pdf/PyPDF2/files/9524160/input.pdf)

Is it possible to get this information for the radio buttons and check boxes?

**Comment by vphan6896** [comment]
I think your input file displays the radio buttons as graphics. I was able to use get_fields() on https://media.wizards.com/2016/dnd/downloads/5E_CharacterSheet_Fillable.pdf and got several fields back.

**Comment by pubpub-zz** [comment]
in general:
get_fields report both the check box status in "/V" (/Off or the name of the active state) for the radio buttons the selected active state is reported too (in this case the radio buttons are in the kids field)

for you specific case, @vphan6896 is right

this issue should be closed

**Comment by pubpub-zz** [comment]
this issue is closed with #1864
